### PR TITLE
mon: show the leader info on mon stat command

### PR DESCRIPTION
**Pull request by songbaisen**
mon: show the leader info on mon stat command

Signed-off-by:song baisen song.baisen@zte.com.cn

### CODE REVIEW

File: src/mon/MonmapMonitor.cc [hunk 1]
```diff
@@ -173,8 +173,9 @@ bool MonmapMonitor::preprocess_command(MonOpRequestRef op)
 
   if (prefix == "mon stat") {
     mon->monmap->print_summary(ss);
-    ss << ", election epoch " << mon->get_epoch() << ", quorum " << mon->get_quorum()
-       << " " << mon->get_quorum_names();
+    ss << ", election epoch " << mon->get_epoch() << ", leader "
```
Comment (tchaikov) [inline src/mon/MonmapMonitor.cc]: the first one in quorum is the leader. so this information is already included in `mon->get_quorum_names()`, right?

Comment (xiexingguo) [inline src/mon/MonmapMonitor.cc]: +1

Comment (songbaisen) [inline src/mon/MonmapMonitor.cc]: @tchaikov  :smile: Yes that is true.But for the people who do not familiar with paxos, he or she may not know that.So make it more clear here for user.

**Comment by dachary** [comment]
jenkins test this please (no logs)

**Comment by liewegas** [comment]
@songbaisen please rebase?  adding leader to the output seems useful for users!

**Comment by songbaisen** [comment]
@liewegas  Ok, I close this and reopen a new one  .https://github.com/ceph/ceph/pull/14177

**Pull request closed without merging**